rkt/cli_apps.go

```diff
@@ -150,6 +155,90 @@ func (ae *appExec) Set(s string) error {
 	return nil
 }
 
+// appInjectVolume is for --inject-volume flags in the form of: --inject-volume source=PATH,target=PATH
+type appInjectVolume apps.Apps
+
+func (aam *appInjectVolume) Set(s string) error {
+	app := (*apps.Apps)(aam).Last()
+	if app == nil {
+		return fmt.Errorf("--inject-volume must follow an image")
+	}
+
+	p := strings.SplitN(s, ":", 2)
+	if len(p) != 2 {
+		return fmt.Errorf("must be SOURCE_PATH:TARGET_PATH")
+	}
+
+	volName, err := types.SanitizeACName(p[1])
+	if err != nil {
+		return fmt.Errorf("must be SOURCE_PATH:TARGET_PATH")
+	}
+
+	// TODO(iaguis) make it a random name?
```

Comment:
I thought there would be problems with different apps having injected volumes with the same target but since "mounts" is per-app, it should be fine. I'll remove the TODO.